testgrid/deploy/tg-script.sh

```diff
@@ -48,22 +49,16 @@ SyslogIdentifier=tgrund
 Environment="KUBECONFIG=/etc/kubernetes/admin.conf"
 Environment="HOME=/root"
 Environment="PATH=/root/.krew/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
-ExecStart=/bin/bash -c 'source /root/.dockerhub-creds; /bin/tgrun run'
+ExecStart=/bin/bash -c '/bin/tgrun run'
 [Install]
 WantedBy=multi-user.target
 TGRUND
 
-cat <<-DHCREDS > /root/.dockerhub-creds
-export DOCKERHUB_PASS="${dh-pass}"
-export DOCKERHUB_USER="${dh-user}"
-export DOCKERHUB_EMAIL="${dh-email}"
-DHCREDS
-
 echo "pulling tgrun image and extracting binary"
 docker pull replicated/tgrun:latest
 
 docker create -ti --name dummy replicated/tgrun:latest bash
 docker cp dummy:/bin/tgrun /bin/tgrun
 docker rm -f dummy
 
-systemctl start tgrun
+# systemctl start tgrun
```

Comment:
I think we still want to start the tgrun service?

Reply:
yeah, forgot to fix after testing